datasafe-rest-impl/2.startDockerImageWithLocalFilesystem.sh

```diff
@@ -0,0 +1,37 @@
+#
+# trap with sigint because container can not be stopped from command line otherwise
+#
+
+trap stop SIGINT
```

Comment:
Instead of trap it is enough to add `-it` to run args

Reply:
yes